Client/Frontend/Reader/ReaderMode.swift

```diff
@@ -11,6 +11,36 @@ enum ReaderModeState: String {
     case Active = "Active"
 }
 
+enum ReaderModeTheme: String {
```

Comment:
Are we planning on having the reader theme controls be native or shown as an element in the page? Android does the latter -- if we followed, then these could all be defined in the JS instead, and the page would just send a JSON string to Swift whenever the user changes prefs (and Swift wouldn't need to actually parse the string).